### PR TITLE
General alcohol tweaks

### LINKED ISSUE #7625: Shotglass becomes regular glass on use
1. Pour drin-FINEST RUSSIAN VODKA MADE ONLY FROM BEST POTATOES AND CAPITALIST TEARS INTO SHOTGLASS

2. Drink... drink.

Your shotglass has now magically grown into a regular-sized glass. Probably did it at the pouring phase, but eh.

Shouldn't shotglasses be a small, set amount, which forces a single pouring and drinking number?

### PR DESCRIPTION
**Pull request by TheVekter**
- Tweaks shot glasses. They can hold 15 units now, so certain shots can be mixed in them without having to use a shaker. Player now drinks all 15 units at once.
- Adds special sprites for rum, vodka, and whiskey (3 most likely boozes for shots). This lets us port certain shot-based drinks (B-52 comes to mind) to shot glass size.
- Ported B-52 and Toxins Special to shot glasses. You can no longer mix them in normal glasses.
- Adds a generic iconstate for everything else.
- Fixes #7625 
- Fixes an unreported bug - apparently I broke the tequila bottle sprite in #7568.

NOTE: The drunkenness tweaks have been removed, as part of #7439 takes care of this issue.

### CODE REVIEW

**Comment by Tenebrosity** [comment]
So this uses different sprites to represent different drinks in the same glass instead of changing the colour of the reagent?

**Comment by TheVekter** [comment]
Yeah. This is pretty similar to how the drinking glass is coded. It's
because there are specific sprites for certain drinks, as opposed to
beakers where it just shows the color of the reagent.

If you pour something like plasma or choral into a drinking glass it shows
up as a glass full of brown liquid and calls it a "glass of what".

File: code/modules/food&drinks/drinks/drinks/drinkingglass.dm [hunk 1]
```diff
@@ -162,6 +162,7 @@
 				icon_state = "tequilasunriseglass"
 				name = "tequila Sunrise"
 				desc = "Oh great, now you feel nostalgic about sunrises back on Terra..."
+			// Toxins Special is also in drinks.dm because shot glasses, mirror any changes there. //
```
Comment (RemieRichards) [inline code/modules/food&drinks/drinks/drinks/drinkingglass.dm]: the first sign that this is not the right way to go about this.

Comment (TheVekter) [inline code/modules/food&drinks/drinks/drinks/drinkingglass.dm]: I'm open to suggestions. I honestly considered removing Toxins Special from glasses entirely, but left it in as a quality of life thing (to not confuse people when their drinks suddenly wouldn't mix).

Comment (RemieRichards) [inline code/modules/food&drinks/drinks/drinks/drinkingglass.dm]: it's less specifically the drinks themselves and the fact that you're having to ask for future changes to be mirrored, hint, nobody is going to do so and if they do then they are a really nice person.

These drinks appear with shotglass icons, now we have actual shotglasses you should just remove the ability to mix said drinks in normal glasses.

Comment (TheVekter) [inline code/modules/food&drinks/drinks/drinks/drinkingglass.dm]: I'll need to make a changelog for it, but alright. I'll keep an eye out and see if anyone adminhelps about their drinks not working.

Comment (RemieRichards) [inline code/modules/food&drinks/drinks/drinks/drinkingglass.dm]: I believe changelogs are becoming more of a mandatory thing so you'll probably need one anyways.
And as with changes it'll probably take a little bit of time for people to learn, a changelog entry would help this.

**Comment by TheVekter** [comment]
I wanna do a bit more testing on this before we merge, but I'm pretty sure this is g2g.

**Comment by TheVekter** [comment]
Alright, looks like everything's working on my end. One last commit and we'll be good.

**Comment by paprka** [comment]
the map edits are fucked in this, you removed the janicart

**Pull request merged**